faiss/IndexRefine.cpp

```diff
@@ -371,8 +371,13 @@ void IndexRefinePanorama::search(
 
     FAISS_THROW_IF_NOT(k > 0);
     FAISS_THROW_IF_NOT(is_trained);
+
+    std::unique_ptr<idx_t[]> del1;
```

Comment:
Similar to how is done [here](https://github.com/facebookresearch/faiss/blob/main/faiss/IndexRefine.cpp#L119-L127).